### PR TITLE
[Nested Lists] Adds className and visual offset for list field for better UX.

### PR DESCRIPTION
**Pull request by jeremyzahner**
## Description of changes

Adds className and visual offset for list field for better UX.

## Related issues (if any)

* #3282 

## Testing

- [x] Please confirm `npm run test-all` ran successfully.

### CODE REVIEW

**Comment by wmertens** [comment]
You don't seem to be following the naming scheme for the class. I'm going to defer to @jossmac @mxstbr who know better what is supposed to happen here.

**Comment by jeremyzahner** [comment]
@wmertens Thanks.

File: fields/types/list/ListField.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ module.exports = Field.create({
 	renderUI () {
 		const { label, value } = this.props;
 		return (
-			<div>
+			<div className="FormFieldList">
 				<h3 data-things="whatever">{label}</h3>
```
Comment (jeremyzahner) [inline fields/types/list/ListField.js]: @JedWatson @wmertens Is "whatever" the correct data-things binding?

Comment (wmertens) [inline fields/types/list/ListField.js]: I do believe that was just for the sake of example, to get something going…

Comment (jeremyzahner) [inline fields/types/list/ListField.js]: Then we probably should rename it to something less "exemplary"? =)

**Pull request merged**